src/java.base/share/classes/java/lang/Runtime.java

```diff
@@ -90,9 +90,10 @@
  *
  * <p>When the JVM terminates, all threads are immediately prevented from executing any further
  * Java code. This includes shutdown hooks as well as daemon and non-daemon threads. The
- * threads' current methods do not complete normally or abruptly; no {@code finally} clause
- * of any method is executed, nor is any {@linkplain Thread.UncaughtExceptionHandler
- * uncaught exception handler}.
+ * threads' current methods do not complete normally or abruptly.
+ * {@linkplain Thread.UncaughtExceptionHandler Uncaught exception handlers} are not run,
+ * no {@code finally} clause of any method is executed, and try-with-resource
```

Comment:
`try-with-resource` or `try-with-resources`?

Reply:
> `try-with-resource` or `try-with-resources`?

Agree should be try-with-resources

Reply:
Yep